chat-components/src/main/java/com/pubnub/components/chat/service/message/action/DefaultMessageReactionServiceImpl.kt

```diff
@@ -21,14 +22,15 @@ import timber.log.Timber
     FlowPreview::class,
     DelicateCoroutinesApi::class,
 )
-class DefaultMessageReactionService(
+class DefaultMessageReactionServiceImpl(
```

Comment:
Is that `Impl` suffix needed?

Reply:
Not really, just a convention. We can remove it everywhere